src/misc/lv_fs.h

```diff
@@ -65,10 +65,12 @@ typedef enum {
     LV_FS_SEEK_SET = 0x00,      /**< Set the position from absolutely (from the start of file)*/
     LV_FS_SEEK_CUR = 0x01,      /**< Set the position from the current position*/
     LV_FS_SEEK_END = 0x02,      /**< Set the position from the end of the file*/
-}lv_fs_whence_t;
+} lv_fs_whence_t;
 
 typedef struct _lv_fs_drv_t {
     char letter;
+    bool enable_cache;
+    uint16_t cache_size;
```

Comment:
Just a thought: perhaps it's better to merge these and use `cache_size == 0` to mean that the cache is disabled? Then `LV_FS_CACHE_DEFAULT_SIZE` wouldn't be needed.

Reply:
I agree.